widget/widget.go

```diff
@@ -99,6 +99,11 @@ func (w *baseWidget) disable(parent fyne.Widget) {
 	canvas.Refresh(parent)
 }
 
+// Disabled returns true if the widget is disabled
+func (w *baseWidget) Disabled() bool {
```

Comment:
We can't expose this at the base level as it will apply to every widget.
Instead this needs to be private and those widgets implementing Disable and Enable should add Disabled.
We can't copy the Visible() pattern as that applied to all widgets